wiki/Organisations/osu!_UCI/fr.md

```diff
@@ -1,36 +1,39 @@
 ---
-outdated: true
 tags:
-  - Californie
```

Comment:
the tags should be things that people using the French locale would search for--- so you probably want French translations of common nouns here